### DISCUSSION TITLE
I want browsers to handle privacy (and GDPR-related) configurations

### DISCUSSION THREAD

**Discussion by aarongustafson** [WebWeWant/webwewant.fyi#87] [Wants]
Currently, websites handle GDPR-settings and info in a ton of different ways. It&#39;s expensive and cumbersome to develop these features again and again and privacy configuration should really be controlled in the user settings of the browser, not on the individual website.

<hr>

https://webwewant.fyi/wants/136/

**Comment by olifink** [comment]
Been thinking along the same lines for a while now. It feels we should be able to do better than have all the notification floats and check boxes for each site. Ideally we would have:

* a declarative way - e.g. a privacy manifest - that describes generically GDPR-settings for a site
* default behaviour can show this to the user as a UI
* browsers should be able to check it or automatically augment it (at least to some degree, e.g. cookies should trigger a cookie info, ads or social should trigger consent info)
* users should be able to set the GDPR-level they are comfortable with as a browser setting and have the browser act as proper agent accepting where appropritate and reporting conflicts
* exceptions should be possible to set for specific sites (ie. like todays security/certificate site-wide exceptions) 
* optionally browser should block interaction outside of users desired privacy scope
* accessibility would be improved
* search engines/SEO ranking should honor sites properly declaring the privacy manifest

if this would be a proper web standard it would make privacy more transparent for users, researchers, developers and browser which don't yet support it could at least use one shim library instead of everyone having to implement it
